### ISSUE TITLE
bloop fish completion prints errors

### ISSUE THREAD

**Issue by ohechtl** [scalacenter/bloop#1479]
Hi!

When I hit <Tab> after inputting an expression like `bloop test ` in the fish terminal, I'll get this text printed before I'll get my completion:
```
bloop test test: Expected a combining operator like '-a' at index 5nTest

/usr/share/fish/vendor_completions.d/bloop.fish (line 105): 
        if test -n $parsed[1] -a -n $parsed[2] -a -n $parsed[3]
           ^
from sourcing file /usr/share/fish/vendor_completions.d/bloop.fish
in command substitution

(Type 'help test' for related documentation)
test: Expected a combining operator like '-a' at index 5

/usr/share/fish/vendor_completions.d/bloop.fish (line 107): 
        else if test -n $parsed[1] -a -z $parsed[2] -a -n $parsed[3]
                ^
from sourcing file /usr/share/fish/vendor_completions.d/bloop.fish
in command substitution

(Type 'help test' for related documentation)
test: Expected a combining operator like '-a' at index 5

/usr/share/fish/vendor_completions.d/bloop.fish (line 109): 
        else if test -n $parsed[1] -a -n $parsed[2] -a -z $parsed[3]
                ^
from sourcing file /usr/share/fish/vendor_completions.d/bloop.fish
in command substitution

(Type 'help test' for related documentation)
test: Expected a combining operator like '-a' at index 5

/usr/share/fish/vendor_completions.d/bloop.fish (line 111): 
        else if test -n $parsed[1] -a -z $parsed[2] -a -z $parsed[3]
                ^
from sourcing file /usr/share/fish/vendor_completions.d/bloop.fish
in command substitution

(Type 'help test' for related documentation)
```
                                                                     
Although I'll get my completion after this, and I won't get a new error if I try to reproduce it again, it's a minor pain point for me.

Do you have any idea what could cause this?

**Comment by tgodzik** [comment]
Thanks for reporting! I am not exactly sure what might be causing these issues. I would welcome any help if anyone is more versed in fish. 

I might take a look into it at some point in the future as I also use fish, but I cannot promise anything.

**Comment by a06kin** [comment]
Looks like fix is available in [v1.4.12](https://github.com/scalacenter/bloop/releases/tag/v1.4.12)
Can we close it @tgodzik?

**Comment by ohechtl** [comment]
Thank you very much! :) :) :)
